src/main/java/com/google/firebase/auth/GetUsersResult.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google Inc.
```

Comment:
LLC

Reply:
I'll leave these instances unchanged for now since they reflect the current state of the master branch. It's best to do a separate PR in the future, and correct all these license headers in one go.